.storybook/webpack.config.js

```diff
@@ -24,7 +57,21 @@ module.exports = (baseConfig) => {
     }
   })
 
-  baseConfig.resolve.modules = [srcPath, 'node_modules']
+  baseConfig.plugins.push(
+    new webpack.DefinePlugin({
```

Comment:
If we use only `DefinePlugin` from webpack, maybe we should destructure `require('webpack')`? That's not necessary, but it may improve readability.